frontend/packages/dev-console/src/components/import/deployImage-validation-utils.ts

```diff
@@ -110,5 +110,6 @@ export const deployValidationSchema = yup.object().shape({
     path: yup
       .string()
       .matches(pathRegex, { message: 'Path must start with /.', excludeEmptyString: true }),
+    unknownTargetPort: yup.number().test(isInteger('Port must be an Integer.')),
```

Comment:
this would not let form  submit be enabled as the default value is `' '` , transform  cann help

```
unknownTargetPort: yup
      .number()
      .transform((cv) => (_.isNaN(cv) ? undefined : cv))
      .test(isInteger('Port must be an Integer.')),
```

Reply:
decided to just go back to my previous solution I had which was "match digits but allow for empty"